tests/testDcrModelGeneration.py

```diff
@@ -132,15 +133,15 @@ def test_calculate_new_model(self):
         # np.save(data_file, (new_solution, inverse_var_arr), allow_pickle=False)
         new_solution_ref, inverse_var_arr_ref = np.load(data_file)
         for soln_new, soln_ref in zip(new_solution, new_solution_ref):
-            self.assertFloatsAlmostEqual(soln_new, soln_ref)
+            self.assertFloatsAlmostEqual(soln_new, soln_ref, printFailures=self.printFailures)
         for inv_var_new, inv_var_ref in zip(inverse_var_arr, inverse_var_arr_ref):
-            self.assertFloatsAlmostEqual(inv_var_new, inv_var_ref)
+            self.assertFloatsAlmostEqual(inv_var_new, inv_var_ref, printFailures=self.printFailures)
 
     def test_calc_model_metric(self):
         """Test that the DCR model convergence metric is calculated consistently."""
         model_file = "test_data/build_model_vals.npy"
-        metric_ref = np.array([0.022835006051, 0.014885210227, 0.0120010522772,
-                               0.0140386530451, 0.0204993350076, 0.0237343323628])
+        metric_ref = np.array([0.0719102938278, 0.0243621751, 0.00996798332164,
+                               0.0235583355852, 0.0309890648232, 0.0339528991862])
```

Comment:
Maybe a short description of where these values are coming from would be in order. As in "This is the expected result from running this part of the DRC pipeline in [this mode]".